pkg/streamingpromql/operators/functions/range_vectors.go

```diff
@@ -874,8 +874,8 @@ func doubleExponentialSmoothing(step *types.RangeVectorStepData, _ float64, args
 			x = smoothingFactor * sample.F
 
 			// Scale the last smoothed value for all samples except the first one in head.
-			if head && i != 0 || !head {
-				estimatedTrend = calcTrendValue(trendFactor, smooth0, smooth1, estimatedTrend)
+			if head && i > 0 || !head {
```

Comment:
[nit] Might be able to simplify this to make this unambiguous:

```suggestion
			if !head || i > 0 {
```